package.json

```
@@ -2,5 +2,25 @@
   "devDependencies": {
     "@types/node": "^22.5.0",
     "ts-node": "^10.9.2"
-  }
+  },
+  "name": "tsrun",
+  "version": "1.0.0",
+  "description": "TSRun is a wrapper script that guesses command fixes using the Gemini API.",
+  "main": "run.ts",
+  "dependencies": {
+    "typescript": "^5.5.4"
```

It's generally better to hard-pin your dependency versions, and update them manually. Otherwise, unexpected things could happen. :)

```suggestion
    "typescript": "5.5.4"
```

---

Done :)